ci: add temp fix to prevent Docker cache from growing

kare opened the pull request:
https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#local-cache

mirpo commented on package.json:
```diff
@@ -40,8 +40,8 @@
   "engines": {
-    "node": "13.11.0",
-    "npm": "6.14.2"
+    "node": "14.7.0",
```
update in github action too please

mirpo replied:
Looks good, let's wait for CI (`npm`) to pass 👍

kare committed:
ci: add temp fix to prevent Docker cache from growing

https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#local-cache
